### ISSUE TITLE
Update design before redeploy from master

### ISSUE THREAD

**Issue by arouinfar** [phetsims/energy-skate-park-basics#473]
Many of the design decisions in Energy Skate Park have affected Energy Skate Park: Basics. Basics was not republished off of master in conjunction with ESP. Before this sim gets republished, we will need to review the current design of ESPB and decide which ESP-only features we would like to add (e.g. skater choice).

**Comment by arouinfar** [comment]
This issue blocks #411

**Comment by arouinfar** [comment]
We should also decide if there are any ESP-only features that would be appropriate to include in the Basics version. For example, the timer and measuring tape may be useful in Basics.

There are also PhET-iO implications to consider. The Studio trees of Geometric Optics and Geometric Optics:  Basics are nearly identical and it's possible to essentially turn one sim into the other using PhET-iO. See https://github.com/phetsims/geometric-optics-basics/issues/3

**Comment by arouinfar** [comment]
This simulation is not currently a priority, unassigning.

**Comment by marlitas** [comment]
@kathy-phet should this design update be included in the upcoming publication?

**Comment by arouinfar** [comment]
@marlitas is the plan to deploy both ESP and ESPB with the ongoing character set work? Currently, only the full ESP has been published with skater localization. This seems like an excellent opportunity to redeploy Basics and modernize it.

Basics is on an extremely stale branch (circa 2016) and completely predates all work on the full ESP sim. During the development of ESP, we made many changes to Basics but these changes have never been tested or deployed. I've detailed the remaining work to be done earlier in this issue, which is nontrivial but lower complexity.

**Comment by marlitas** [comment]
Yes. As I understand it ESPB would be redeployed with the new region and culture skaters as well. I'm not sure how much time @kathy-phet wants to put into the redesign, but I assume if there are straightforward aspects it would be beneficial to include them in the redeploy.

**Comment by marlitas** [comment]
@amanda-phet is the designer I've been working with on this to provide feedback and design support. It might be useful to set up a meeting with the three of us to determine what could be included in this upcoming ESPB publication?

**Comment by arouinfar** [comment]
@marlitas @amanda-phet I think it would be good for the three of us to meet and go over ESPB. There are many unpublished changes in that sim, so I don't think it's a good candidate for localization unless we can spend a little time on cleanup too.

**Comment by marlitas** [comment]
@arouinfar, @kathy-phet let me know that we should not focus on localization for this sim with so many things up in the air. I've pushed the priority down for this sim, and I imagine it will not be part of the upcoming publications because of that.
